### PR TITLE
add turborepo

### PR DESCRIPTION
**Pull request by akellbl4**
- adds turborepo for package managment
- move lint-staged (it can work by package)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,7 +17,7 @@ debug.test
 remark42
 /bin/
 /backend/var/
-/backend/app/var/
+/var/
```
Comment (paskal) [inline .gitignore:20]: That change doesn't seem right.

File: compose-dev-frontend.yml [hunk 2]
```diff
@@ -13,8 +13,8 @@ services:
       dockerfile: Dockerfile
       args:
         - SKIP_BACKEND_TEST=true
-        - SKIP_FRONTEND_BUILD=true
-        # - NODE_ENV=development
+        # - SKIP_FRONTEND_BUILD=true
```
Comment (paskal) [inline compose-dev-frontend.yml:16]: I think skipping frontend build should be enabled by default here, it's done that way to reduce the build time significantly.

Comment (akellbl4) [reply compose-dev-frontend.yml]: dropped this change

File: .gitignore (hunk 1 above)
Comment (akellbl4) [reply .gitignore]: dropped this change

**Review by paskal — APPROVED** [review]

**Review by Mavrin — APPROVED** [review]

**Pull request merged**